tests/unit/sagemaker/serve/model_server/multi_model_server/test_multi_model_server_prepare.py

```diff
@@ -12,13 +12,64 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+from pathlib import PosixPath
+import platform
 from unittest import TestCase
 from unittest.mock import Mock, patch
 
+import numpy as np
+
 from sagemaker.serve.model_server.multi_model_server.prepare import _create_dir_structure
 
+from sagemaker.serve.model_server.multi_model_server.server import (
+    LocalMultiModelServer,
+)
+
+CPU_TF_IMAGE = "763104351884.dkr.ecr.us-east-1.amazonaws.com/huggingface-pytorch-inference:2.0.0-transformers4.28.1-cpu-py310-ubuntu20.04"
+MODEL_PATH = "model_path"
+MODEL_REPO = f"{MODEL_PATH}/1"
+ENV_VAR = {"KEY": "VALUE"}
+PAYLOAD = np.random.rand(3, 4).astype(dtype=np.float32)
+DTYPE = "TYPE_FP32"
+SECRET_KEY = "secret_key"
+INFER_RESPONSE = {"outputs": [{"name": "output_name"}]}
+
 
 class MultiModelServerPrepareTests(TestCase):
+    def test_start_invoke_destroy_local_multi_model_server(self):
```

Comment:
I think we need to re-run this locally to understand where it is failing

Reply:
Got it